src/goose/cli/session.py

```diff
@@ -70,10 +73,9 @@ def __init__(
         self.prompt_session = GoosePromptSession()
         self.status_indicator = Status("", spinner="dots")
         self.notifier = SessionNotifier(self.status_indicator)
-
+        langfuse_context.configure(enabled=tracing)
```

Comment:
this line outputs:
```
Langfuse client is disabled. No observability data will be sent.
```

when langfuse is not setup (e.g. no --tracing) - should probably call this only if tracing is True?

Reply:
it might happen that this line is logged once when the sdk is loaded but tracing is disabled.

@hassiebp did we already or will we change this behavior?

Reply:
The observe_wrapper is imported in all the exchange providers with a completion function before `langfuse_context.configure(enabled=tracing)` is even called - so langfuse is actually initialized before the tracing flag is parsed if there is an available langfuse context (ie you are running local langfuse). So configuring the context to be disabled is required if that's the case to disable the tracing.

Reply:
I've updated it so that the log level is set to error and the message is suppressed in case --tracing is not set. Also updated some prints for if tracing is enabled but langfuse is not available.

@baxen  should we exit if --tracing was passed but langfuse isn't available on start up? Or just allow the session to continue running?

Reply:
decided to exit if --tracing is passed but langfuse not available